### DISCUSSION TITLE
Are you planning to port LibreSprite to Android?

### DISCUSSION THREAD

**Discussion by ElBlandersito** [LibreSprite/LibreSprite#447] [Q&A]
Hello, as a regular Android user, I would like to know if you have plans to port LibreSprite to that platform.

**Comment by pegvin** [comment]
It's HIGHLY unlikely that would happen, LibreSprite's codebase is a mess to manage, and that's why `Dotto` is in development.

And Yes, Dotto will be available on Android.

**Reply by timothyparez** [reply]
There is/are android build(s) out there.
I actually have one that works, but I cannot remember where I got it.

filename:  app-all_in_one-debug.apk
md5: 0606e5690e16998e8d03dba4c20f5ae7 

![image](https://github.com/LibreSprite/LibreSprite/assets/2060470/2d960d41-266b-4113-91a1-5299b9259cd6)

This is a screenshot of it running on Android and it works great.  
If only I could remember where I got it.

**Reply by ElBlandersito** [reply]
Are you sure you didn't get it on some website?

**Reply by jjhaggar** [reply]
You can download the Android version for LibreSprite from this same repo (1.1-dev Preview): https://github.com/LibreSprite/LibreSprite/releases
![image](https://github.com/LibreSprite/LibreSprite/assets/4078392/1536792d-e252-41a5-a3f2-e1f110274b3b)

**Comment by timothyparez** [comment]
@jjhaggar Thank you, yes, that's where I got it. I couldn't remember because by default the `Assets` expansion box is 
closed for preview versions.

![image](https://github.com/LibreSprite/LibreSprite/assets/2060470/aa297f0a-5b19-47ad-8f21-1d6afa4c0457)

@ElBlandersito You can try that version, it works great for me on some tablets.
